lib/bait.js

```diff
@@ -43,7 +48,18 @@ var Bait = {
                 return prev.then(function(result) {
                     return curr.bind(self)(result);
                 });
-            }, startingArgs);
+            }, startingArgs)
+            .tap(function(args){
+                laterArray.reduce(function(prev, curr) {
+                    // call the current fn with the previous fn's return value
+                    // and returns that as a promise
+                    return prev.then(function(result) {
+                        return curr.bind(self)(result);
+                    });
+                }, args))
+                .then(function(){ return null;})
+                .catch(errorHandler);
```

Comment:
I don't see how this would work this way.

I imagine something like:
```javascript
laterArray.reduce(function(prev, curr) {
    // call the current fn with the previous fn's return value
    // and returns that as a promise
    return prev.then(function(result) {
        return curr.bind(self)(result);
    });
}, [clone of args]))
.then(function(){ return null;})
.catch(errorHandler);

return args;
```